src/gui/Src/Gui/MainWindow.cpp

```diff
@@ -1090,6 +1093,31 @@ void MainWindow::updateWindowTitleSlot(QString filename)
     }
 }
 
+void MainWindow::updateDarkTitleBar()
+{
+    typedef void(* RTLGETNTVERSIONNUMBERS)(DWORD * MajorVersion, DWORD * MinorVersion, DWORD * BuildNumber);
+    static auto RtlGetNtVersionNumbers = (RTLGETNTVERSIONNUMBERS)GetProcAddress(GetModuleHandleW(L"ntdll.dll"), "RtlGetNtVersionNumbers");
+    DWORD major, minor, build;
+    RtlGetNtVersionNumbers(&major, &minor, &build);
+
+    if(major < 10 || build < 17763)

```

Comment:
What if `major > 10`? [Bad version number checks
](https://devblogs.microsoft.com/oldnewthing/20040213-00/?p=40633)

Reply:
Microsoft has indicated that Windows 10 will be the last major version ever released so I don't think this is too much of an issue. That said, if it can be improved, how would you write the check?

Reply:
`if(major < 10 || (major == 10 && build < 17763))`

Reply:
Thanks!